watertap/unit_models/ion_exchange_0D.py

```diff
@@ -425,117 +563,25 @@ def build(self):
             doc="Number of operational columns for ion exchange process",
         )
 
-        self.number_columns_redund = Var(
-            initialize=1,
-            bounds=(1, None),
-            units=pyunits.dimensionless,
-            doc="Number of redundant columns for ion exchange process",
-        )
-
-        # ====== Kinetic variables ====== #
-
-        self.partition_ratio = Var(
-            initialize=100,
-            bounds=(0, None),
-            units=pyunits.dimensionless,
-            doc="Partition ratio",
-        )
-
-        self.fluid_mass_transfer_coeff = Var(
-            self.target_ion_set,
-            initialize=1e-3,
-            bounds=(0, None),
-            units=pyunits.m / pyunits.s,
-            doc="Fluid mass transfer coefficient",
-        )
-
-        self.rate_coeff = Var(
-            self.target_ion_set,
-            initialize=2e-4,
-            bounds=(0, 1),
-            units=pyunits.m**3 / (pyunits.kg * pyunits.s),
-            doc="Rate coefficient",
-        )
-
         self.t_breakthru = Var(
             initialize=1e5,  # DOW, ~7 weeks max breakthru time
             bounds=(0, None),
             units=pyunits.s,
             doc="Breakthrough time",
         )
 
-        self.t_cycle = Var(
-            initialize=1e5,
-            bounds=(0, None),
-            units=pyunits.s,
-            doc="Cycle time",
-        )
-
         self.t_contact = Var(
-            initialize=520,
-            bounds=(120, None),
+            initialize=120,
+            bounds=(100, None),
             units=pyunits.s,
-            doc="Contact time",
+            doc="Resin contact time",
         )
 
-        self.t_waste = Var(
-            initialize=1800,
-            bounds=(1, None),
+        self.ebct = Var(
+            initialize=520,
+            bounds=(90, None),
```

Comment:
Does this lower bound impact model stability?

Reply:
I don't think so (but I haven't really checked), this lower bound is more about having a lower limit on the EBCT so that when optimizing, the model doesn't force water through the column as fast as possible, which would break some of the assumptions of the modeling approach.